src/model-helpers/DocumentList.js

```diff
@@ -0,0 +1,97 @@
+import {
+  getSolidDataset,
+  setThing,
+  getThingAll,
+  saveSolidDatasetAt,
+  createSolidDataset,
+  saveFileInContainer,
+  getThing,
+  removeThing,
+  deleteFile
+} from '@inrupt/solid-client';
+
+import { makeDocIntoThing, parseDocFromThing } from './Document';
+
+const parseFromDataset = async (docsDataset, session) => {
+  const docList = [];
+  const allDocThings = getThingAll(docsDataset);
+  await Promise.all(
+    allDocThings.map(async (userThing) => {
+      const docObject = await parseDocFromThing(userThing, session);
+      docList.push(docObject);
+    })
+  );
+
+  return docList;
+};
+
+const saveToPod = async (session, { dataset, containerUrl }) => {
+  const newDataset = await saveSolidDatasetAt(`${containerUrl}doclist.ttl`, dataset, {
+    fetch: session.fetch
+  });
+  const newList = await parseFromDataset(newDataset, session);
```

Comment:
parseFromDataset() is taking in session as a parameter, but parseDocFromThing() doesn't utilize it. Can be removed along with the await.

Reply:
fixed